examples/ShonanAveragingCLI.cpp

```diff
@@ -58,6 +61,10 @@ int main(int argc, char* argv[]) {
       "Write solution to the specified file")(
       "dimension,d", po::value<int>(&d)->default_value(3),
       "Optimize over 2D or 3D rotations")(
+      "useHuberLoss,h", po::value<bool>(&useHuberLoss)->default_value(false),
+	  "set True to use Huber loss")(
+	  "pMin,p", po::value<int>(&pMin)->default_value(3),
```

Comment:
spacing. Please format (only parts that have been changed to avoid large diff) and also lint